openfasoc/generators/cryo-gen/simulation/templates/cryoInst_ngspice.sp

```diff
@@ -1,25 +1,28 @@
-
-.lib '@@PATH_TO_LIB' tt
-.include '@@PATH_TO_DUT_SP'
-.include '@@PATH_TO_SC_SP'
-
-.param vvdd = 1.8
-
-xi1 EBL OUT VDD VSS cryoInst
-
-vEBL EBL 0 pwl 0 0 10n 0 '10n+1f' 'vvdd'
-vVDD VDD 0 dc 'vvdd'
-vVSS VSS 0 dc 0
-
-c0 OUT 0 1f
-
-.TRAN 50p 100n
-
-.control
-run
-set filetype=ascii
-exit
-
-.endc
-
+
+.lib '@@PATH_TO_LIB' tt
+.include '@@PATH_TO_DUT_SP'
+.include '@@PATH_TO_SC_SP'
```

Comment:
@Jerry-lqx Can you please the @@PATH_TO_SC_SP variable. While running the flow it's not updated for final spice file 
![image](https://user-images.githubusercontent.com/61928883/230597784-270b19fd-6eb5-42ad-be6e-48f7d15f04e3.png)

The simulation is stopped because of this.

![image](https://user-images.githubusercontent.com/61928883/230598003-2427edd7-0aeb-4645-b14b-edaa76e3f735.png)

Reply:
I have updated the file.